GuitarString.cpp

```diff
@@ -0,0 +1,132 @@
+/******************************************************************
+    GuitarString.cpp
+
+    implementation of the GuitarString structure
+******************************************************************/
+
+#include "GuitarString.h"
+
+char GuitarString::getOpenString(int strNum)
+{
+    switch (strNum)
+    {
+        case 1:
+            return 'E';
+        case 2:
+            return 'B';
+        case 3:
+            return 'G';
+        case 4:
+            return 'D';
+        case 5:
+            return 'A';
+        case 6:
+            return 'E';
+        default:
+            return 'E';
+    }
+}
+
+GuitarString::GuitarString()
+{
+    //std::cout << "GuitarString default CTOR" << std::endl;
+    numFrets = NUM_FRETS;
+    stringNum = 1;
+    openStringKey = 'E';
+    frettedKey = 'E';
+    frettedNum = 0;
+    note.setStringNum(1);
+    note.setFretNum(frettedNum);
+    note.setFinger(none);
+    note.setKey('E');
+}
+
+GuitarString::GuitarString(int strNum)
+{
+    //std::cout << "GuitarString CTOR" << std::endl;
+    numFrets = NUM_FRETS;
+    stringNum = strNum;
+    openStringKey = getOpenString(strNum);
+    frettedKey = openStringKey;
+    frettedNum = 0;
+
+    note.setStringNum(strNum);
+    note.setFretNum(frettedNum);
+    note.setFinger(none);
+    note.setKey(openStringKey);
+}
+
```

Comment:
You have a lot of repetition here.
come up with a separate function that initializes your properties and call it from both constructors
- [ ] revise.

Reply:
Added initializeString() member function done.